CMakeLists.txt

```diff
@@ -418,7 +418,8 @@ option(WITH_RADOSGW "Rados Gateway is enabled" ON)
 option(WITH_RADOSGW_BEAST_OPENSSL "Rados Gateway's Beast frontend uses OpenSSL" ON)
 option(WITH_RADOSGW_AMQP_ENDPOINT "Rados Gateway's pubsub support for AMQP push endpoint" ON)
 option(WITH_RADOSGW_KAFKA_ENDPOINT "Rados Gateway's pubsub support for Kafka push endpoint" ON)
-option(WITH_RADOSGW_LUA_PACKAGES "Rados Gateway's support for dynamically adding lua packagess" ON)
+option(WITH_RADOSGW_LUA "Rados Gateway's support for Lua" ON)
+option(WITH_RADOSGW_LUA_PACKAGES "Rados Gateway's support for dynamically adding lua packagess" ${WITH_RADOSGW_LUA})
```

Comment:
i think `cmake_dependent_option()` is what you want for this one, see https://cmake.org/cmake/help/latest/module/CMakeDependentOption.html

Reply:
Thanks for pointing that out. I was not aware this existed, but it is indeed what I want.